Description: Patch nlmake Makefile for Debian builds.
 - Change to use the standard flag LDFLAGS instead of using LFLAGS.
 - Set CFLAGS using '+=' instead of just '=' to enable compile time changes.
Author: Robert James Clay <jame@rocasa.us>
Origin: vendor
Last-Update: 2013-06-08

--- nlmake-1.13.1.2+030617.orig/src/Makefile
+++ nlmake-1.13.1.2+030617/src/Makefile
@@ -2,8 +2,8 @@ SRC_DIR=./
 OBJ=.o
 EXE=
 GPP=g++
-CFLAGS=-g -Wall -pedantic
-LFLAGS=
+CFLAGS+= -g -Wall -pedantic
+#LFLAGS=
 
 
 CDEFS= -DINCS_NEED_DOT_H -DNOEXCEPTIONS $(ADDCDEFS)
@@ -34,10 +34,10 @@ default: nlmake$(EXE)
 all: nlmake$(EXE)
 
 nlmake$(EXE): $(OBJFILES)
-	$(GPP) $(LFLAGS) -o nlmake$(EXE) $(OBJFILES)
+	$(GPP) $(LDFLAGS) -o nlmake$(EXE) $(OBJFILES)
 
 install:
-	$(INSTALL) $(IBOPT) nlmake$(EXE) $(BINDIR)
+	$(INSTALL) $(IBOPT) nlmake$(EXE) $(DESTDIR)$(DIRSEP)$(BINDIR)
 
 uninstall:
 	-$(RM) $(RMOPT) $(BINDIR)$(DIRSEP)nlmake$(EXE)
